<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pay</class>
 <widget class="QMainWindow" name="pay">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>500</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>481</width>
      <height>100</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>500</width>
      <height>100</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
width: 144px;
height: 50px;
opacity: 1;
font-size: 36px;
font-family: Roboto, Roboto-Bold;
font-weight: Bold;
text-align: left;
color: #556f59;
line-height: 0px;
}</string>
    </property>
    <property name="text">
     <string>结算</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>481</width>
      <height>300</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>500</width>
      <height>300</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget{
width: 414px;
height: 223px;
opacity: 1;
background: #6d8b72;
font-size: 7px;
font-family: Microsoft YaHei, Microsoft YaHei-Normal;
font-weight: Normal;
text-align: center;
color: #333333;
line-height: 10px;
text-shadow: 0px 3px 6px 0px rgba(0,0,0,0.15); 
}</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QWidget" name="widget_2" native="true">
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>40</y>
          <width>90</width>
          <height>25</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel{
width: 90px;
height: 50px;
opacity: 1;
font-size: 18px;
font-family: Microsoft YaHei, Microsoft YaHei-Bold;
font-weight: Bold;
text-align: left;
color: #fafafa;
line-height: 50px;
}</string>
        </property>
        <property name="text">
         <string>起点：</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>30</y>
          <width>151</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel{
width: 191px;
height: 39px;
opacity: 1;
font-size: 28px;
font-family: Microsoft YaHei, Microsoft YaHei-Bold;
font-weight: Bold;
text-align: left;
color: #fafafa;
line-height: 0px;
border-width:0;
border-style:outset;
background-color:transparent;

}</string>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="widget_3" native="true">
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>40</y>
          <width>81</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel{
width: 90px;
height: 50px;
opacity: 1;
font-size: 18px;
font-family: Microsoft YaHei, Microsoft YaHei-Bold;
font-weight: Bold;
text-align: left;
color: #fafafa;
line-height: 50px;
}</string>
        </property>
        <property name="text">
         <string>目的地：</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>30</y>
          <width>141</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel{
width: 191px;
height: 39px;
opacity: 1;
font-size: 28px;
font-family: Microsoft YaHei, Microsoft YaHei-Bold;
font-weight: Bold;
text-align: left;
color: #fafafa;
line-height: 0px;
border-width:0;
border-style:outset;
background-color:transparent;

}</string>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="widget_4" native="true">
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>40</y>
          <width>72</width>
          <height>25</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel{
width: 90px;
height: 50px;
opacity: 1;
font-size: 18px;
font-family: Microsoft YaHei, Microsoft YaHei-Bold;
font-weight: Bold;
text-align: left;
color: #fafafa;
line-height: 50px;
}</string>
        </property>
        <property name="text">
         <string>总距离：</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>30</y>
          <width>61</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel{
width: 191px;
height: 39px;
opacity: 1;
font-size: 28px;
font-family: Microsoft YaHei, Microsoft YaHei-Bold;
font-weight: Bold;
text-align: left;
color: #fafafa;
line-height: 0px;
border-width:0;
border-style:outset;
background-color:transparent;

}</string>
        </property>
        <property name="text">
         <string>公里</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>30</y>
          <width>51</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel{
width: 191px;
height: 39px;
opacity: 1;
font-size: 28px;
font-family: Microsoft YaHei, Microsoft YaHei-Bold;
font-weight: Bold;
text-align: left;
color: #fafafa;
line-height: 0px;
border-width:0;
border-style:outset;
background-color:transparent;

}</string>
        </property>
        <property name="text">
         <string>1.5</string>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>420</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushutton{
width: 195px;
height: 50px;
opacity: 1;
background: #eeeeee;
font-size: 18px;
font-family: Microsoft YaHei, Microsoft YaHei-Bold;
font-weight: Bold;
text-align: center;
color: #ff001d;
line-height: 21px;
text-shadow: 0px 3px 6px 0px rgba(0,0,0,0.15); 
}</string>
    </property>
    <property name="text">
     <string>支付：25元</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
